controllers/node_controller_test.go

```diff
@@ -44,6 +44,10 @@ var _ = Describe("Node controller reconcile", func() {
 				Name: existingNodeName,
 			},
 		}
+		expectRequeueAfterIsSet = func(result ctrl.Result) {
+			ExpectWithOffset(1, result.RequeueAfter).To(BeNumerically(">=", time.Minute))
+			ExpectWithOffset(1, result.RequeueAfter).To(BeNumerically("<=", float64(time.Minute)+(float64(time.Minute)*0.1)))
```

Comment:
I think it would be safer to use variable for the jitter value, if we decided to increase the jitter interval, then the test may 'get lucky' and flake later randomly.